frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/status-card.tsx

```diff
@@ -27,30 +27,32 @@ import { getCephHealthState, getDataResiliencyState } from './utils';
 const cephStatusQuery = STORAGE_HEALTH_QUERIES[StorageDashboardQuery.CEPH_STATUS_QUERY];
 const resiliencyProgressQuery = DATA_RESILIENCY_QUERY[StorageDashboardQuery.RESILIENCY_PROGRESS];
 
-const CephAlerts = withDashboardResources(({ watchAlerts, stopWatchAlerts, alertsResults }) => {
-  React.useEffect(() => {
-    watchAlerts();
-    return () => {
-      stopWatchAlerts();
-    };
-  }, [watchAlerts, stopWatchAlerts]);
+export const CephAlerts = withDashboardResources(
```

Comment:
I think these changes are just a re-format, but I'm OK with that :smiley: 

(It happened to me that saving files after the recent ESLint tech bump caused a re-format as well.)